### ISSUE TITLE
API v1

### ISSUE THREAD

**Issue by pinootto** [FlavioF/karaf-cellar-kubernetes-example#2]
Hello,

I tried the example. There is a problem with the API version. Currently kubernetes changed the API to v1. The example works with API v1beta1, which is not compatible with v1.

So we need to change the file .yaml according to the new API version.

https://github.com/kubernetes/kubernetes/blob/master/docs/api.md#v1beta3-conversion-tips

Can you help me to write the new .yaml files according to the new API v1?

**Comment by FlavioF** [comment]
Yes, ofc. However I could help only in a couple of days.. But it is easy. 
Do you made any try to convert it? If yes, Please let me take a look into it.
